SimFastTiming/Configuration/python/SimFastTiming_cff.py

```diff
@@ -3,3 +3,7 @@
 from SimFastTiming.FastTimingCommon.fastTimeDigitizer_cfi import *
 
 from SimFastTiming.FastTimingCommon.mtdDigitizer_cfi import *
+from SimFastTiming.FastTimingCommon.mtdDigitizer_cfi import _barrel_bar_MTDDigitizer
+
+from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
+phase2_timing_layer_bar.toModify(mtdDigitizer, barrelDigitizer = _barrel_bar_MTDDigitizer)
```

Comment:
I'd suggest
```suggestion
phase2_timing_layer_bar.toModify(mtdDigitizer, barrelDigitizer = _barrel_bar_MTDDigitizer.clone())
```

Otherwise e.g. doing the following here
```python
mtdDigitizer.barrelDigitizer.maxSimHitsAccTime = 200
```
would change the `SimFastTiming.FastTimingCommon.mtdDigitizer_cfi._barrel_bar_MTDDigitizer.maxSimHitsAccTime` as well (since they would be the same objects), which, if done, could be surprising and cause some heavy debugging.

Reply:
I modified the file as you suggest.